recipes/glaze/all/conanfile.py

```diff
@@ -0,0 +1,77 @@
+from conan import ConanFile
+from conan.errors import ConanInvalidConfiguration
+from conan.tools.files import get, copy
+from conan.tools.build import check_min_cppstd
+from conan.tools.scm import Version
+from conan.tools.layout import basic_layout
+import os
+
+required_conan_version = ">=1.50.0"
+
+class PackageConan(ConanFile):
+    name = "glaze"
+    description = "Extremely fast, in memory, JSON and interface library for modern C++"
+    license = "MIT"
+    url = "https://github.com/conan-io/conan-center-index"
+    homepage = "https://github.com/stephenberry/glaze"
+    topics = ("json", "memory", "header-only")
+    settings = "os", "arch", "compiler", "build_type"
+    no_copy_source = True
+
+    @property
+    def _minimum_cpp_standard(self):
+        return 20
+
+    @property
+    def _compilers_minimum_version(self):
+        return {
+            "Visual Studio": "16",
+            "msvc": "192",
+            "gcc": "11",
+            "clang": "12",
+        }
+
+    def layout(self):
+        basic_layout(self, src_folder="src")
+
+    def requirements(self):
+        self.requires("fmt/9.1.0")
+        self.requires("fast_float/3.5.1")
+        self.requires("frozen/1.1.1")
+        self.requires("nanorange/20200505")
+
+    def package_id(self):
+        self.info.clear()
+
+    def validate(self):
+        if self.settings.compiler == "apple-clang":
+            raise ConanInvalidConfiguration(f"{self.ref} requires C++{self._minimum_cpp_standard} and does not support {self.settings.compiler}.")
```

Comment:
According to https://en.cppreference.com/w/cpp/compiler_support, concept library is available since apple-clang 13.1

Reply:
Thank you for your information!
I am going to fix and try it.

Reply:
I confirmed it on apple-clang 14.0.
Thanks!